kernel/build/build_deb

```diff
@@ -88,11 +88,17 @@ echo "*****"
 # Clean the git repo
 git clean -fxd
 
+# Install build dependencies
+sudo apt-get install -y bc
+
 # Prep the config
 echo "Preparing the kernel config"
-cp "$(dirname $0)/kernel-config-deb.basic" .config
+wget https://raw.githubusercontent.com/ceph/ceph-build/master/kernel/build/kernel-config-deb.basic -O .config
```

Comment:
Well, there is one other way of achieving this and it is as simple as making the config file a shell script, i.e. adding
`cat << EOF > file` 
at the beginning of the file,
`EOF`
at the end of the file and calling the script in the builders.

In fact, in my case, I think this might be the best way to achieve this. It adds only a minor overhead (few bytes for cat and EOF bits), it is tracked in scm and you can modify the config files quite easily in the Jenkins UI if needed.